readme.md

```diff
@@ -129,8 +129,8 @@ Rest Variables are not to changes
     about - About the bot
     status - Status of all the downloads
     server - Get server status
-    usettings - User Settings
-    settings - Settings of the bot ⚠️ Admin Only
+    settings - User Settings
+    manage - manage bot settings ⚠️ Admin Only
```

Comment:
i have different plans for this.

Reply:
there is a provision to change these commands if the user likes by setting the ENV vars.